### PR TITLE
Clean up default log printing to prevent clutter

### PR DESCRIPTION
**Pull request by LeoNatan**
This will be the default output:
Good

Instead of:
Bad

### CODE REVIEW

File: detox/src/utils/customConsoleLogger.js [hunk 1]
```diff
@@ -46,9 +46,9 @@ function overrideConsoleMethods(console, bunyanLogger) {
   if (!console.__detox_log__) {
     const log = bunyanLogger;
 
-    override(console, 'log', log.info.bind(log));
+    override(console, 'log', log.debug.bind(log));
```
Comment (noomorph) [inline detox/src/utils/customConsoleLogger.js:49]: Not entirely sure if this is a legit change. If users want `console.log` they should get console.log.

Comment (noomorph) [reply detox/src/utils/customConsoleLogger.js:49]: It is absolutely their problem in their code if this clutters things.

Comment (LeoNatan) [reply detox/src/utils/customConsoleLogger.js]: OK, that makes sense. Will revert this.

Comment (LeoNatan) [reply detox/src/utils/customConsoleLogger.js]: Ah cool, you reverted.

**Comment by LeoNatan** [comment]
Let me encode the gif, and then merge.

**Pull request merged**